landscape/lib/format.py

```diff
@@ -0,0 +1,31 @@
+import inspect
+
+
+def format_object(object):
```

Comment:
I was going to complain about shadowing builtin object, but I see this is verbatim of existing code… carry on.